From 77ffc8ce76b0de8ef36e51b3b0e5ffc3a9bb7a88 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 26 Jan 2015 10:24:31 +0000
Subject: [PATCH 2/2] MinGW-w64: 64-bit ptr_intT fix

Change #if _MSC_VER to #if defined(_WIN32) since
the former works for Visual Studio only whereas
the latter also works for GCC and Clang.
---
 src/libqhull/mem.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libqhull/mem.h b/src/libqhull/mem.h
index ad89de8..857aa5f 100644
--- a/src/libqhull/mem.h
+++ b/src/libqhull/mem.h
@@ -78,7 +78,7 @@ Trace short and quick memory allocations at T5
     Qhull uses int instead of size_t except for system calls such as malloc, qsort, qh_malloc, etc.
     This matches Qt convention and is easier to work with.  
 */
-#if _MSC_VER && defined(_WIN64)
+#if defined(_WIN32) && defined(_WIN64)
 typedef long long ptr_intT;
 #else
 typedef long ptr_intT;
-- 
2.2.2

